metrics/deploy/rbac.yaml

```diff
@@ -73,6 +73,14 @@ rules:
     - get
     - list
     - watch
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - clusterversions
+  verbs:
+    - get
+    - list
+    - watch
```

Comment:
These are still not added to the other file.

Reply:
I don't think so, already added to exporter-role.yaml